src/java.base/share/classes/jdk/crac/impl/GlobalContext.java

```diff
@@ -0,0 +1,19 @@
+package jdk.crac.impl;
+
+import jdk.crac.Context;
+import jdk.crac.Resource;
+import sun.security.action.GetPropertyAction;
+
+public class GlobalContext {
+    private static final String GLOBAL_CONTEXT_IMPL_PROP = "jdk.crac.globalContext.impl";
```

Comment:
As mentioned in off-GH conversation, I don't think this option would be ever used, and if you are going to change the default I'd rather drop it to keep the code simple.

Reply:
Since there is no way to employ BlockingContext other than as the global context implementation, I prefer to make it selectable.

Reply:
OK, so in the future you intend to just change the default and provide non-blocking implementation as the workaround for cases where the registration is not synchronized properly? In that case it makes sense...

Reply:
Exactly. At some point we'll set on a single, the best implementation. But after the experience, I think we'll need the configuration for a while even after we change the default.